README.md

```diff
@@ -23,81 +23,108 @@ Alternatively  look  here for a [RAML definition](raml/file-upload.raml)
 ### Envelope
 
 #### Create An Envelope
-Creates an envelope and auto generates an Id. Successful response provides in the headers the link  and the envelope Id.
+Creates an envelope and auto generates an Id. The body in the http request must be json. Successful response provides in the headers the link and the envelope Id.
```

Comment:
We could be more explicit here and say: Response will include a Location header with a link to the newly created envelope.